### PR TITLE
added functionality to install the VSCode context menus.

### PR DESCRIPTION
**Pull request by detlefs**
added functionality to install the VSCode context menus. Also, VSCode is now always added to the search path

### CODE REVIEW

**Comment by tylerl0706** [comment]
This is great! Thanks for contributing 😃 

FYI, we got another PR that touched this file so their may be a very small merge conflict coming your way. Changelog stuff.

When you get a chance, sign the CLA so we can get this in 👍

**Comment by tylerl0706** [comment]
@detlefs did you want to resolve the conflict so we can get this in :)

**Comment by rjmholt** [comment]
Although this is stale, this would still be nice to have. Maybe we could resolve the conflicts ourselves and merge in?

File: scripts/Install-VSCode.ps1 [hunk 1]
```diff
@@ -202,4 +213,4 @@ if (!($IsLinux -or $IsOSX)) {
 }
 else {
     Write-Error "This script is currently only supported on the Windows operating system."
-}
+}
```
Comment (rkeithhill) [inline scripts/Install-VSCode.ps1]: Should probably add back in the final newline.

**Comment by tylerl0706** [comment]
Ooo need to add this to the signing process so we can release this on the Gallery

**Pull request merged**